### ISSUE TITLE
should pubsub-remove-subscribers actually be pubsub-remove-subscriber ? 

### ISSUE THREAD

**Issue by ksk438** [dask/distributed#8334]
https://github.com/dask/distributed/blob/0dc9e8800d6e49e77e2fd5cbc71f0033cccf7b3b/distributed/pubsub.py#L188

I don't see an operation called "pubsub-remove-subscribers" .... only "pubsub-remove-subscriber". I have seen a related error in the past but don't have it on hand. Can anyone comment? 

I have patched it in my local copy to be the latter and it has been working well.

**Comment by fjetter** [comment]
Yes, this appears to be a typo. Looks like this is untested. Are you interested in contributing a patch (with a test)?

**Comment by ksk438** [comment]
I don't have the capacity at the moment. If nobody picks it up I could give it a shot in a couple months I believe.

**Comment by killcoder** [comment]
Still not patched

dask                      2024.11.2
distributed            2024.11.2

```
distributed\core.py", line 913, in handle_stream
    handler = self.stream_handlers[op]
              ~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'pubsub-remove-subscribers'
```

**Comment by hendrikmakait** [comment]
@killcoder: A fix is welcome. As a heads-up: The pub-sub functionality has already been deprecated.

**Comment by killcoder** [comment]
@hendrikmakait thanks for the response, I managed to figure out the replacement subscriber_topic / log_event
